benchmarks/functional_autograd_benchmark/vision_models.py

```diff
@@ -0,0 +1,491 @@
+import torch
+from torch import nn
+from torch.nn import functional as F
+import torchvision_models as models
+
+from utils import make_functional, load_weights
+from scipy.optimize import linear_sum_assignment
+
+def get_resnet18(device):
+    N = 10
+    model = models.resnet18(pretrained=False)
+    criterion = torch.nn.CrossEntropyLoss()
+    model.to(device)
+    params, names = make_functional(model)
+
+    inputs = torch.rand([N, 3, 224, 224], device=device)
+    labels = torch.rand(N, device=device).mul(10).long()
+
+    def forward(*new_params):
+        load_weights(model, names, new_params)
+        out = model(inputs)
+
+        loss = criterion(out, labels)
+        return loss
+
+    return forward, params
+
+def get_fcn_resnet(device):
+    N = 10
+    criterion = torch.nn.MSELoss()
+    model = models.fcn_resnet50(pretrained=False, pretrained_backbone=False)
+    model.to(device)
+    params, names = make_functional(model)
+
+    inputs = torch.rand([N, 3, 224, 224], device=device)
```

Comment:
We use slightly larger input sizes for the segmentation models, `3 x 480 x 480` is a standard size